modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -571,6 +562,13 @@ protected void validateField(
 		DDMFormEvaluatorFieldContextKey ddmFormEvaluatorFieldContextKey =
 			entry.getKey();
 
+		if (filterVisibleFieldsWithConfirmationField(
```

Comment:
@carolmariaabb, can we rename the method `filterVisibleFieldsWithConfirmationField`?

Invoke this method on a stream it's ok, but when we invoke it on a `if` statement then its name may not be the best choice. How about `isFieldWithConfirmationFieldVisible`?

Reply:
I fully agree :D